### DISCUSSION TITLE
Something like Index Cards?

### DISCUSSION THREAD

**Discussion by Valdus92** [vkbo/novelWriter#1402] [Ideas]
I know some of those other apps have it, nameless as they shall be. But I was wondering if seeing the tree in index card form is possible or if there is a workaround?

**Comment by vkbo** [comment]
This is discussed in feature #70, and what we ended up agreeing on was the current synopsis feature in combination with the outline view.

I do plan to add more types of outline views, and one that shows boxes with scene information is one idea. These will likely not be interactive views though. The core idea of novelWriter is that all data is contained within the text documents themselves, aside from the structure defining bits which live in the main project XML file. The second principle that comes into play here is that the only place this content can be changed is in the main document editor panel. An interactive index card feature will break both these design principles, so we're left with alternative display solutions. Various ideas on this are discussed in #76 and #854.

**Reply by Valdus92** [reply]
After spending sometime with novelwriter I have seen the error of my ways. It's something totally different, and I appreciate and enjoy it very much.

**Reply by vkbo** [reply]
Heh, well, it is intended to be different but also be flexible enough to not lock you too much into a specific way of structuring your project. I never liked the index card approach, and neither was it a topic when I initially started planning out novelWriter with a colleague back in 2018. The design choices are therefore somewhat orthogonal to that way of thinking.
